lib/facebook/graph.ex

```diff
@@ -48,6 +48,14 @@ defmodule Facebook.Graph do
     request(:get, url, options)
   end
 
+  @doc """
+  HTTP GET using a full URL and options
+  """
+  @spec getFullURL(url, options) :: response
+  def getFullURL(url, options \\ []) do
```

Comment:
I thought of that, but I didn't want to expose to much of the Graph module.  But I think it's the better choice.